### PR TITLE
Add leadership PACs to about this candidate section of candidate profile pages

### LINKED ISSUE #4270: Add leadership PACs to about this candidate section of candidate profile pages
**What we're after:**
We need to add leadership PACs to applicable candidate profile pages. Specifically at the bottom of the about this candidate section. Leadership PAC names should link to those committee profile pages, just as they do for all other committees listed in that Committees section. 

### Related issues
- https://github.com/fecgov/fec-cms/issues/4253 Design for the inclusion of leadership PACs on sponsoring candidate profile pages

### Action items
- [x] Clickable links to leadership PACs are added to applicable candidate profile page "about this candidate" sections
- [x] Glossary links are added to each of the committee subheaders: 
  - Authorized campaign committees: Authorized committee 
  - Joint fundraising committees: Joint fundraising committee
  - Sponsored leadership PACs: Leadership PAC

### Completion criteria
- [x] Leadership PAC committee(s) appear on the candidate profile page under the "About this candidate" tab.
- [x] Open follow-up ticket to add links from committee profile pages to candidate profile pages. Currently it's only name and no link to profile page.

### Design
Screen Shot 2020-12-17 at 4 05 36 PM

### PR DESCRIPTION
**Pull request by fec-jli**
## Summary (required)
Add leadership PACs to the committees section in 'about this candidate' page

- Resolves #4270

## Impacted areas of the application
about this candidat section in Candidate Profile page

List general components of the application that this PR will affect:
1)Add leadership PAC to committees section
2)Add Glossary links to each of the committee subheaders

## Screenshots
Screen Shot 2021-02-01 at 2 37 50 PM

## How to test
1)checkout branch
npm run build
2)run tests:
pytest
./manage.py test
./manage.py test data.tests
3)compare some examples with prod

(1)P00009092 / 2020  (only have one dsgn=P committee)
http://127.0.0.1:8000/data/candidate/P00009092/?tab=about-candidate
https://www.fec.gov/data/candidate/P00009092/?tab=about-candidate

(2)S4AR00103 /2020 (has P, A, 2J and two leadership pac committees)
http://127.0.0.1:8000/data/candidate/S4AR00103/?tab=about-candidate&cycle=2020&election_full=true
https://www.fec.gov/data/candidate/S4AR00103/?tab=about-candidate&cycle=2020&election_full=true

(3)H0MI08042 / 2014 (only has two leadership pac committees)
http://127.0.0.1:8000/data/candidate/H0MI08042/?tab=about-candidate
https://www.fec.gov/data/candidate/H0MI08042/?tab=about-candidate

(4)P00009183 / 2020  (display only one leadership pac committee)
http://127.0.0.1:8000/data/candidate/P00009183/?tab=about-candidate&election_full=true
https://www.fec.gov/data/candidate/P00009183/?tab=about-candidate

(5)H0OH08029 / 2016(only has one leadership pac committee)
http://127.0.0.1:8000/data/candidate/H0OH08029/?tab=about-candidate
https://www.fec.gov/data/candidate/H0OH08029/?tab=about-candidate

(6)click Glossary links

### CODE REVIEW

**Comment by fec-jli** [comment]
Note:
1)The candidate(id=P00009183) returns two rows from api result,
 one is pcc converted to D, another is leadership pac committee,
remove the duplicate committees with same committees_id.
api call: https://fec-dev-api.app.cloud.gov/v1/candidate/P00009183/committees/history/2020/

**Before remove the duplicate committees:**

before_remove_duplicate

**After remove the duplicate committees:**

after_remove_duplicate

The code to pick up distinct committees in data/view.py(start from line 205)
`len_d_cmte = len(committees_d)`
        `committees_leadership_pac.append(committees_d[0])`
       ` for i in range(len_d_cmte):`
           ` if (i + 1) < len_d_cmte and committees_d[i].get("committee_id") != committees_d[i + 1].get("committee_id"):`
               `committees_leadership_pac.append(committees_d[i + 1])`

2)add new sub-class in _glossary.scss:
 `&.term-inline {
      display: inline-block;
    }
`

File: fec/fec/static/scss/components/_glossary.scss [hunk 1]
```diff
@@ -95,6 +95,9 @@
   margin-right: 2px;
   padding-right: .9em;
 
+  &.term-inline {
+      display: inline-block;
```
Comment (johnnyporkchops) [inline fec/fec/static/scss/components/_glossary.scss:99]: @fec-jli , You should add `margin-bottom u(1rem)` to match the original vertical spacing  on these heading tags

**Review by johnnyporkchops — CHANGES_REQUESTED** [review]

**Comment by JonellaCulmer** [comment]
@fec-jli Thanks for catching that duplicate committee! This looks really good. 

Would it be possible to move the glossary link + icon before the ":" instead of after?
cc: @dorothyyeager

File: fec/data/templates/partials/candidate/about-candidate.jinja [hunk 2]
```diff
@@ -93,26 +96,36 @@
...
-      {% if committee_groups['J'] %}
-
-      <h4>Joint fundraising committees:</h4>
-
+      {% if committee_groups['P'] and committee_groups['D'] %}
+      <hr class="hr--lighter">
+      {% endif %}
+      
+      {% if committees_leadership_pac %}    
+      <h4 class="callout__title term term-inline" data-term="Leadership PAC">Sponsored Leadership PACs:</h4>
```
Comment (patphongs) [inline fec/data/templates/partials/candidate/about-candidate.jinja:118]: The `L` in Leadership should be lowercase. It should look like this: `Sponsored leadership PACs`

**Review by patphongs — CHANGES_REQUESTED** [review]
Thanks @fec-jli, this looks really good. Just once change from me to update the casing of the sponsored leadership PACs heading

**Comment by johnnyporkchops** [comment]
> @fec-jli Thanks for catching that duplicate committee! This looks really good.
> 
> Would it be possible to move the glossary link + icon before the ":" instead of after?
> cc: @dorothyyeager

@JonellaCulmer , @fec-jli  and I paired on this and although its possible with an :`after` style, it requires removing the colon from the actual text. We agreed this would be a 508  issue to avoid. So we leave the colon before the icon for now.

**Review by patphongs — APPROVED** [review]
Changes look good, thanks @fec-jli

**Review by johnnyporkchops — APPROVED** [review]
Looks great Jun, thanks for taking time to get CSS right!

**Review by JonellaCulmer — APPROVED** [review]
Looks/works great @fec-jli! Thanks for your hard work on this.

**Comment by fec-jli** [comment]
Thanks Paul correct the data on two tables:
disclosure.cand_cmte_linkage_alternate
disclosure.cand_cmte_linkage 
after today’s materialized view refreshment, I check the front-end on dev, It looks good.
the two examples below, candidate name is a little bit difference:)
a)ROGERS, MICHAEL J — H0MI08042 / C00370791
https://dev.fec.gov/data/candidate/H0MI08042/?tab=about-candidate
https://dev.fec.gov/data/committees/?sponsor_candidate_id=H0MI08042
b)ROGERS, MICHAEL — H2AL03032 / C00439521
https://dev.fec.gov/data/candidate/H2AL03032/?tab=about-candidate
https://dev.fec.gov/data/committees/?sponsor_candidate_id=H2AL03032

**Pull request merged**